index.html

```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
-
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <title>Первая страница</title>
+  </head>
+  <body>
+    <h1>"Моя первая страница"</h1>
+  <p><a href="./about/">"Об авторе"</a></p>
```

Comment:
![image](https://user-images.githubusercontent.com/79144475/224276952-d872edbc-eec0-4bd8-9739-4021c5703bd4.png)

Reply:
You need to add full address here

Reply:
I changed the address